locales/en-US/strings.properties

```diff
@@ -102,6 +102,9 @@ prefs_topstories_description2=Great content from around the web, personalized fo
 prefs_topstories_options_sponsored_label=Sponsored Stories
 prefs_topstories_sponsored_learn_more=Learn more
 prefs_highlights_description=A selection of sites that you’ve saved or visited
+prefs_highlights_options_visited_label=Visited Pages
+prefs_highlights_options_download_label=Most Recent Download
```

Comment:
Question from a localizer: should this be downloads (plural)?

Reply:
This is singular as it shows only one download

Reply:
Ah, thanks! (cc @ TonnesM)

Reply:
Yeah, I noticed by testing meanwhile but thanks for the reminder. ;)

Side note: we used l10n for "Most recently downloaded" which a) covers both singular and plural (at least for my locale), and b) circumvents the use of "download" as a noun. Note that some locales (de, es) currently do use translations for "downloads" (noun, plural).